bin/config-validator.rb

```diff
@@ -340,7 +340,7 @@ def check_rm_variables(manifest)
   end
 
   manifest['configuration']['variables'].each do |variable|
-    next if Common.special_use(variable['name'])
+    next if variable['internal']
```

Comment:
I think we keep the comment from the `special_use` function here. Something like

```
# "internal" variables are defined but not used in the role manifest. They are referenced directly in scripts.
```